yast2_firewall_allowed-services-20170209.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "area": [
+    {
+      "xpos": 139,
+      "ypos": 127,
+      "width": 128,
+      "height": 21,
+      "type": "match"
+    }
+  ],
+  "tags": [
+    "yast2_firewall_allowed-services"

```

Comment:
Yes, this page is about Allowed Services, but HTTP Server is added which needs to checked:
http://e13.suse.de/tests/1820#step/yast2_firewall/12